RELEASE-NOTES.md

```diff
@@ -38,6 +54,13 @@ This major release introduces breaking changes:
 - Download/upgrade your solr version to 9.5.0
   - Solr upgrade is not supported for 3.0.0 with old cores, you must start with a new core (new solr-home)
     - Please use a new Solr home and reindex all objects due to incompatibility between old data and the new Solr schema.
+    - Example below to reindex all objects
+      ```sh
+      # curl -X PUT -H "Authorization: Bearer $TOKEN" https://<your-host>/<your-context>/d1/mn/v2/index?all=true
+      # where $TOKEN is an environment variable containing your administrator jwt token
```

Comment:
this mention of `your administrator jwt token` could also cross-reference the token section I described in the note on line 35. Maybe we should keep the naming consistent, and instead of `your administrator jwt token`, this should say `your indexer token (see section whatever)`??

Reply:
I have updated the wording here, and added a line to see overview to get more info about the indexer token.